index.html

```diff
@@ -1,9 +1,15 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>JavaScript Foundations</title>
-  </head>
-  <body></body>
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>JavaScript Foundations</title>
+</head>
+<body>
```

Comment:
- [ ] Move scripts to head and defer

- [ ] Add content to body. Otherwise Netlify will not run any deploy in preview environment.